### ISSUE TITLE
VoiceOver Support

### ISSUE THREAD

**Issue by atg** [atg/chocolat-public#1015]
Received via email:

> As promised on Twitter, here are the issues that I'm seeing with Chocolat and VoiceOver.
> 
> Before I start: if you're not familiar with it, you can start and stop VoiceOver by pressing Cmd+F5. To navigate and hear the different controls, you use:
> - Ctrl+Option+Left and Right arrow to go left and right,
> - Ctrl+Option+DownArrow to interact (zoom into) a container such as a tool bar, HTML container etc.
> - Ctrl+Option+Up Arrow to stop that interaction again.
> 1. The order of controls is very unusual for a COCOA app. It starts with the Close button, but then goes on with an unlabeled button (where VoiceOver just says "Button"), followed by Minimize,, an unlabeled checkbox, the Zoom button, the correctly labeled "Find" textbox, two unabeled buttons, the document/window title, the Replace textfield, the Replace button, the "14 Days left" button, Replace All button, two unlabeled buttons, and the full screen button.
>    Then, there's the scroll area with a ruler and the main text area inside, which work fine.
> 
> Following the scroll area are 4 popup buttons. None of them have descriptions. The first three show values like "plain text", "4" and "no symbols".
> 
> The buttons are usually not labeled if they are captioned through images rather than plain text. One normally has to add alternative text for those image buttons in these cases for VoiceOver to know what they do.
> 1. The AutoComplete doesn't read with VoiceOver. To reproduce, just provoke the autoComplete to come up. VoiceOver will say "Chocolate has new window", but when switching to that window, it has no content that is accessible to VoiceOver.
> 
> This is probably the harder part to get working, but I've known of AutoComplete solutions that work on OS X.
